CHANGES.md

```diff
@@ -15,6 +15,7 @@
 
 #### Fixed
 
+- Migrate to OCaml 4.08 AST to add support for `let*` bindings (#190, @gpetiot)
```

Comment:
I'd move that to `Added`, since this adds support for 4.08 within code blocks and thus should require a minor version bump, not just a patch.

Reply:
done